mqtt/mqtt3/examples/publisher.rs

```diff
@@ -103,11 +103,11 @@ fn main() {
         .shutdown_handle()
         .expect("couldn't get shutdown handle");
     runtime.spawn(async move {
-        let () = tokio::signal::ctrl_c()
```

Comment:
:(

Reply:
Clippy was super opinionated about this. But... I agree with you :)